src/components/DropList/DropList.jsx

```diff
@@ -219,15 +212,32 @@ function DropListManager({
     }
   }
 
-  function toggleOpenedState(shouldOpen, origin = '') {
-    setOpenOrigin(origin)
+  function toggleTippy(instance, visible) {
+    if (instance == null || instance.state.isDestroyed) return
+
+    if (visible) {
+      instance.show()
+    } else {
+      instance.hide()
+    }
+  }
+
+  function toggleOpenedState(shouldOpen) {
     setOpenedState(shouldOpen)
     onOpenedStateChange(shouldOpen)
+    toggleTippy(tippyInstance, shouldOpen)
   }
 
   return (
     <Tippy
       {...tippyProps}
+      onCreate={instance => {
+        setTippyInstance(instance)
+        getTippyInstance(instance)
```

Comment:
Can you explain why there's `set` and `get` called right after each other? That's probably fine, but I am just curious what it does here :)

Reply:
I removed this part, now using a `ref` instead of storing the tippy instance in state

Reply:
FYI `getTippyInstance` is a callback prop, in case you want to do something with it from the outside